main/etc/systemd/system/cockpit.service

```diff
@@ -1,15 +1,34 @@
 [Unit]
-Description=quay.io/cockpit/ws:latest podman container
+Description=Cockpit Container
 After=network-online.target
 Wants=network-online.target
+SourcePath=/etc/containers/systemd/cockpit.container
```

Comment:
Tested and this `SourcePath=` line needs to go away. It's technically wrong now that we removed the container file, but more importantly, it results in systemd constantly warning.

```
systemctl status cockpit
Warning: The unit file, source configuration file or drop-ins of cockpit.service changed on disk. Run 'systemctl daemon-reload' to reload units.
```

Reply:
i pushed this fix.  It does start up in my vm and can auto-update though the logs do throw some errors on the container spinning up:

```
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + mkdir -p /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + chmod 755 /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + chown root:root /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + mkdir -p /etc/ssh
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + '[' podman = oci -o podman = podman ']'
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + '[' -n '' ']'
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + /bin/mount --bind /host/etc/cockpit /etc/cockpit
May 16 03:44:45 coreos-cockpit-02 ws[3019]: + /usr/libexec/cockpit-certificate-ensure
May 16 03:44:45 coreos-cockpit-02 ws[3019]: '/etc/os-release' -> '/host/etc/os-release'
May 16 03:44:45 coreos-cockpit-02 ws[3019]: '/usr/lib/os-release' -> '/host/usr/lib/os-release'
```

which seem to be due to the startup script in "/container/label-run" inside the container

Reply:
Yep, i see the same on my VM. I don't think it's errors, just verbose `bash -x` type output.